DigitalLearningSolutions.Web/Helpers/CommonValidationErrorMessages.cs

```diff
@@ -15,5 +15,6 @@ public static class CommonValidationErrorMessages
         public const string PasswordRequired = "Enter a password";
         public const string PasswordMinLength = "Password must be 8 characters or more";
         public const string PasswordMaxLength = "Password must be 100 characters or fewer";
+        public const string StringMaxLengthValidation = "{0} cannot exceed {1} characters";

```

Comment:
I have updated other constants but we could also make these more generic but I chose not to do for this task unless we need to?

Reply:
Sorry, I should have been more clear. I had meant that only the StringMaxLengthValidation needed its wording to be consistent with the other messages. Not that the others need changing.

I do see your point though. In the past, we have had similar refactoring done that just popped up in tickets. However, this is not a major refactor that will affect much so it can be left for now.

I'll leave the decision to you. You could leave the other messages how they were. Or, if you think it will be quick and easy to refactor and consolidate the other Max length messages and confirm that they work properly, feel free to make those changes here and I'll look over them again. Though if you do refactor the other messages, it will be worth mentioning in the ticket which messages were affected so they can be tested.

Reply:
Sorry, my understanding was incorrect. I think refactoring requires additional testing and I don't think it is very important as you have mentioned. I have left other messages as they were before, and updated only StringMaxLengthValidation as suggested.